composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
+        "satooshi/php-coveralls": "^0.7.1"
```

Comment:
@SpacePossum This could be indeed an issue. But could be.

This is why I said on https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1995#discussion_r67324037:

> If **we are able to** not make it dependents, don't do it.

BTW, a WIP is here concerning this issue: https://github.com/sebastianbergmann/phpunit/issues/2015